server/routes/utilMongoose.js

```diff
@@ -193,33 +200,49 @@ router.get('/seed/', async (req, res, next) => {
         
         // create 10 companies
         for (let i = 0; i < COMPANY_COUNT; i++) {
+            let companyName = companies[i]
```

Comment:
kinda looks to me like the uuidv4 wrapper could be its own function to simplify the changes to this code

Reply:
2 liner, not much of a gain there?